### PR TITLE
Added documentation for configuring Killbill to send events to AWS SQS.

### PR DESCRIPTION
**Pull request by vnandwana**

### CODE REVIEW

File: userguide/tutorials/events-to-aws-sqs.adoc [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+== Overview
+
+This document outlines the steps to configure the Killbill application to send events to an Amazon Simple Queue Service (SQS) queue.
+
+== Prerequisites
+
+* Access to AWS Management Console.
+* IAM permissions to create and manage SQS and IAM roles.
+* SSH access to the EC2 instance running Killbill.
```
Comment (reshmabidikar) [inline userguide/tutorials/events-to-aws-sqs.adoc]:
```suggestion
* SSH access to the EC2 instance running Kill Bill.
```

File: userguide/tutorials/events-to-aws-sqs.adoc [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+7. Click on *Next: Tags*, add any tags if needed, and then click on *Next: Review*.
+
+8. Enter a name for the role and click on *Create role*.
+
+=== Step 3: Attach the IAM Role to the EC2 Instance
+
+1. Navigate to the EC2 Dashboard in the AWS Management Console.
+
+2. Select the EC2 instance where Killbill is running.
```
Comment (reshmabidikar) [inline userguide/tutorials/events-to-aws-sqs.adoc]:
```suggestion
2. Select the EC2 instance where Kill Bill is running.
```

File: userguide/tutorials/events-to-aws-sqs.adoc [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+3. Click on *Actions*, then *Security*, and choose *Modify IAM role*.
+
+4. Select the IAM role created in Step 2 from the dropdown menu.
+
+5. Click on *Update IAM role*.
+
+=== Step 4: Configure Killbill to Use SQS
+
+1. SSH into the EC2 instance running Killbill.
```
Comment (reshmabidikar) [inline userguide/tutorials/events-to-aws-sqs.adoc]:
```suggestion
1. SSH to the EC2 instance running Kill Bill.
```

File: userguide/tutorials/events-to-aws-sqs.adoc [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+Restart the EC2 instance to apply the changes:
+
+[source]
+----
+sudo reboot
+----
+
+=== Step 6: Test the Configuration
+
+1. To test the setup, install any plugin, add a user, or make any tenant configuration changes in Killbill. These actions should generate events that will be sent to the SQS queue.
```
Comment (reshmabidikar) [inline userguide/tutorials/events-to-aws-sqs.adoc]:
```suggestion
1. To test the setup, install any plugin, add a user, or make tenant configuration changes in Kill Bill. These actions should generate events that will be sent to the SQS queue.
```

File: userguide/tutorials/events-to-aws-sqs.adoc [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    "eventType": "TENANT_CONFIG_CHANGE",
+    "accountId": null,
+    "tenantId": "4effbb8c-1c40-4051-871a-613ff6e954cb",
+    "metaData": "{\"key\":\"CATALOG\"}"
+  }
+----
+
+== Conclusion
+
+By following these steps, you should be able to configure Killbill to send events to an AWS SQS queue, enabling better event-driven integrations and workflows.
```
Comment (reshmabidikar) [inline userguide/tutorials/events-to-aws-sqs.adoc]:
```suggestion
By following these steps, you can configure Kill Bill to send events to an AWS SQS queue, enabling better event-driven integrations and workflows.
```

File: userguide/tutorials/events-to-aws-sqs.adoc [hunk 6]
```diff
@@ -0,0 +1,117 @@
+= Configuring Killbill to Send Events to AWS SQS
+
+== Overview
+
+This document outlines the steps to configure the Killbill application to send events to an Amazon Simple Queue Service (SQS) queue.
```
Comment (reshmabidikar) [inline userguide/tutorials/events-to-aws-sqs.adoc]: Maybe add a link to https://docs.killbill.io/latest/kill_bill_events to explain what a Kill Bill event is?

File: userguide/tutorials/events-to-aws-sqs.adoc [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+2. Edit the `/var/lib/tomcat/bin/setenv2.sh` file to include the following properties. Be sure to replace `queueName` with the name of your SQS queue and adjust `maxRetries` if necessary:
+
+[source]
+----
+-Dcom.killbill.aws.enableSqsEvents=true
+-Dcom.killbill.aws.sqs.queueName=<queue-name>
+-Dcom.killbill.aws.sqs.maxRetries=5
+----
+
+=== Step 5: Restart the Killbill EC2 Instance
```
Comment (reshmabidikar) [inline userguide/tutorials/events-to-aws-sqs.adoc]:
```suggestion
=== Step 5: Restart the Kill Bill EC2 Instance
```

**Review by reshmabidikar — COMMENTED** [review]
Looks good, I've left some minor comments.

**Comment by vnandwana** [comment]
> Looks good, I've left some minor comments.

Thank you, Reshma for these comments. I have addressed them, please check when you get a chance.

**Review by reshmabidikar — APPROVED** [review]

**Pull request merged**